Update ROCm-DS to AMD Data Science (#30)
ROCm-DS/ROCm-DS: RELEASE.md

Reviewer: ```suggestion
- [AMD Data Science Release highlights](#amd-data-science-release-highlights)
```

Reviewer: ```suggestion
- [AMD Data Science components](#amd-data-science-components)
```

Reviewer: ```suggestion
## AMD Data Science release highlights
```

> AMD is pleased to announce the ROCm Data Science Toolkit (ROCm-DS) 26.03 release with major component upgrades, broader hardware enablement, and new distributed computing capabilities for AMD GPU environments. ROCm-DS is an open-source data science toolkit designed to improve the performance of data preparation, analytics, machine learning, and vector search workloads on AMD GPUs. 

Reviewer: ```suggestion
AMD is pleased to announce the AMD Data Science Toolkit 26.03 release with major component upgrades, broader hardware enablement, and new distributed computing capabilities for AMD GPU environments. AMD Data Science is an open-source data science toolkit designed to improve the performance of data preparation, analytics, machine learning, and vector search workloads on AMD GPUs. 
```

> With the 26.03 release, ROCm-DS continues to strengthen its software stack while improving portability for users bringing existing GPU data science workflows to AMD hardware. The 26.03 release introduces dask-hip and hip-ucxx as new components, adding foundational support for distributed and multi-GPU workflows on ROCm. In addition, core libraries, including hipDF, hipMM, hipRAFT, and hipVS, have been updated to incorporate newer upstream functionality and support for ROCm 7.2.3. 

Reviewer: ```suggestion
With the 26.03 release, AMD Data Science continues to strengthen its software stack while improving portability for users bringing existing GPU data science workflows to AMD hardware. The 26.03 release introduces dask-hip and hip-ucxx as new components, adding foundational support for distributed and multi-GPU workflows on ROCm. In addition, core libraries, including hipDF, hipMM, hipRAFT, and hipVS, have been updated to incorporate newer upstream functionality and support for ROCm 7.2.3. 
```

```{note}

> hipGRAPH remains early access (EA) in ROCm-DS 26.03. 

Reviewer: ```suggestion
hipGRAPH remains early access (EA) in AMD Data Science 26.03. 
```

* Major hipMM upgrade to 4.0.0, introducing a precompiled shared library model (`librmm.so`), improved logging integration, enhanced diagnostics, and updated Python packaging. 
* hipRAFT 1.0.0 and hipVS 1.0.0 updated with new primitives, performance improvements, API changes, and expanded multi-GPU capabilities. 
* Expanded hardware support, including support for the gfx950 GPU architecture across major libraries, with additional experimental gfx11xx/gfx12xx RDNA support in hipDF. 
* ROCm 7.2.3 support across updated components. 
* Python 3.13 support added in selected components. 
* Continued support for Python and C++ APIs, helping developers integrate ROCm-DS into analytics, AI, and GPU-accelerated data science workflows. 

> This release advances ROCm-DS toward a more complete platform for distributed, multi-GPU, and large-scale data science workloads on AMD GPUs. 

Reviewer: ```suggestion
This release advances AMD Data Science toward a more complete platform for distributed, multi-GPU, and large-scale data science workloads on AMD GPUs. 
```

## System requirements

> For the 26.03 release, the ROCm-DS components support the ROCm 7.2.3 release. Refer to the {doc}`Compatibility matrix <compatibility-matrix>` to verify supported AMD Instinct GPUs, operating systems, and ROCm versions.

Reviewer: ```suggestion
For the 26.03 release, the AMD Data Science components support the ROCm 7.2.3 release. Refer to the {doc}`Compatibility matrix <compatibility-matrix>` to verify supported AMD Instinct GPUs, operating systems, and ROCm versions.
```

> ## ROCm-DS components

Reviewer: ```suggestion
## AMD Data Science components
```
